src/pcre2test.c

```diff
@@ -3217,6 +3244,84 @@ return i+1;
 }
 
 
+#ifdef SUPPORT_PCRE2_16
+/*************************************************
+*      Convert UTF-16 character to code point     *
+*************************************************/
+
+/* This function reads one or more UTF-16 code units, and returns the
+codepoint of that character.
+
+Argument:
+  utf16units  a pointer to the units vector
+  end         a pointer to the end of the units vector
+  vptr        a pointer to an int to receive the value
+
+Returns:      > 0  => the number of 16-bit units consumed
+              -1   => malformed UTF-16
+*/
+
+static int
+utf16_to_ord(PCRE2_SPTR16 utf16units, PCRE2_SPTR16 end, uint32_t *vptr)
+{
+uint32_t c = *utf16units++;
+
+if (c >= 0xDC00 && c <= 0xDFFF) return -1;
```

Comment:
Do we use uppercase for hex?

Reply:
Apparently we do. I just extracted this code into a function, from elsewhere in the file.

Reply:
Lowercase is far more common throughout PCRE2. I've reformatted these few uppercase ones here for consistency.